backend/src/main/java/com/redhat/cloud/notifications/routers/NotificationResource.java

```diff
@@ -275,11 +275,21 @@ public Response updateBehaviorGroup(@Context SecurityContext sec,
         String orgId = getOrgId(sec);
 
         if (request.displayName != null) {
+            UUID bundleId = behaviorGroupRepository.getBundleId(id);
             BehaviorGroup behaviorGroup = new BehaviorGroup();
             behaviorGroup.setId(id);
             behaviorGroup.setDisplayName(request.displayName);
+            behaviorGroup.setBundleId(bundleId);
+
+            boolean didUpdate;
+            try {
+                didUpdate = behaviorGroupRepository.update(orgId, behaviorGroup);
+            } catch (BadRequestException badRequestException) {
+                // The duplicate name in bundle id validation failed - keep the responses consistent by returning false.
```

Comment:
Which responses?

We are returning a 400 status with explanation in case of creation failure because of a duplicate name. Shouldn't we do the same for the update?

Reply:
Before this PR, the update failure returns a 200 with a `false` in the body - The only usual way AFAIK to trigger this status is to send a behavior group id of other org id.

I agree that `false` does not really provide any information - what do you think about changing to throwing bad request in both cases?